src/muqsit/vanillagenerator/generator/nether/NetherGenerator.php

```diff
@@ -41,9 +41,9 @@ private static function densityHash(int $i, int $j, int $k) : int{
 	/** @var int */
 	protected $bedrock_roughness = 5;
 
-	public function __construct(ChunkManager $world, int $seed, array $options = []){
-		parent::__construct($world, $seed, Environment::NETHER, null, $options);
-		$this->addPopulators(new NetherPopulator($world->getWorldHeight()));
+	public function __construct($seed, array $options = []){
+		parent::__construct($seed, Environment::NETHER, null, $options);
+		$this->addPopulators(new NetherPopulator($this->getWorldHeight())); // This isn't faithful to original code. Was $world->getWorldHeight()
```

Comment:
Why was this changed?

Reply:
This was changed because otherwise it doesn't extend VanillaGenerator properly.
[https://github.com/adams-pm-plugins/VanillaGenerator/blob/611ab10950975da9ebec1b8af6c1223dcccea725/src/muqsit/vanillagenerator/generator/VanillaGenerator.php#L43](url)

Reply:
Hmm alright, while at it you may as well remove https://github.com/adams-pm-plugins/VanillaGenerator/blob/master/src/muqsit/vanillagenerator/generator/VanillaGenerator.php#L19-#L32